components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -297,31 +299,32 @@ void CosmeticFiltersJSHandler::OnRemoteDisconnect() {
   EnsureConnected();
 }
 
-void CosmeticFiltersJSHandler::ProcessURL(const GURL& url,
-                                          base::OnceClosure callback) {
+bool CosmeticFiltersJSHandler::ProcessURL(const GURL& url) {
```

Comment:
we need a histogram for this method as discussed in DM

Reply:
added (below the initial early returns as per @atuchin-m's suggestion)